getting-started/README.md

```diff
@@ -1,71 +1,111 @@
-    "Client": "NameOfYourCompanyOrApplication"
+  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+  "AccessToken": "7059D2C25BF64EA681ACAB3A00B859CC-D91BFF2B1E3047A3E0DEC1D57BE1382",
+  "Client": "NameOfYourCompanyOrApplication"
 }
 ```
 
-If the [response](../guidelines/responses.md) looks something like this, you've successfully made your first request!
-```javascript
+- **Expected response**  
+If successful, the API will return property details:
+```json
 {
-    "NowUtc": "2021-05-05T11:39:29Z",
-    "Enterprise": {
-        "Id": "c65ea6e9-2340-42f4-9136-ab3a00b6da22",
-        "ChainId": "8ddea57b-6a5c-4eec-8c4c-24467dce118e",
-        "CreatedUtc": "2020-01-06T11:05:44Z",
-        "Name": "Connector API Hotel (Net Pricing) TEST",
-        "TimeZoneIdentifier": "America/New_York",
-        "LegalEnvironmentCode": "US-DC",
-        "AccommodationEnvironmentCode": "US",
-        "AccountingEnvironmentCode": "DEFAULT",
-        "TaxEnvironmentCode": "US-DC",
-        "DefaultLanguageCode": "en-US",
-        "EditableHistoryInterval": "P0M2DT0H0M0S",
-        "WebsiteUrl": "https://www.mews.com/",
-        "Email": "charging-api@mews.lixx",
-        "Phone": "+12025555501xx",
-        "LogoImageId": "c90a8d7e-a332-45c0-ac44-ac5600a4116e",
-        "CoverImageId": "c867f682-4003-4518-b96c-abec0090a23b",
-        "Address": {},
-        "Currencies": [],
-        "Pricing": "Net",
-        "TaxPrecision": null
-    },
-    "PaymentCardStorage": {
-        "PublicKey": "1100016827"
-    }
+  "NowUtc": "2021-05-05T11:39:29Z",
+  "Enterprise": {
+    "Id": "c65ea6e9-2340-42f4-9136-ab3a00b6da22",
+    "Name": "Connector API Hotel (Net Pricing) TEST",
+    "TimeZoneIdentifier": "America/New_York",
+    "LegalEnvironmentCode": "US-DC",
+    "AccommodationEnvironmentCode": "US",
+    ...
+  },
+  "PaymentCardStorage": {
+    "PublicKey": "1100016827"
+  }
 }
 ```
 
-## Next steps
+> **Requests and responses**: For more on request formatting and response codes, see [Requests](../guidelines/requests.md) and [Responses](../guidelines/responses.md) respectively.
+
+## Step 3: Learn the API essentials
+
+### Time and data serialization
+- All dates and times are in UTC.
+- Dates, durations and other data types follow specific [serialization rules](../guidelines/serialization.md).
 
-There are several [serialization](../guidelines/serialization.md) patterns that you should get familiar with in order to use the API correctly. For example, we work with times in UTC so you should be aware of the timezone the property is in, as well as any daylight saving time changes, and adjust accordingly.
+### Pagination
+- Some operations (like [Get all reservations](../operations/reservations.md#get-all-reservations-2023-06-06)) use [pagination](../guidelines/pagination.md) to handle large result sets.
+
+### Rate limits
+- Shared demo credentials may return `429` errors if multiple developers are testing the same property.
```

Comment:
```suggestion
- Shared demo credentials may return `429 - Too many requests` response code if multiple developers are testing the same property.
```
nitpick: to me reading it, it felt like it's gonna return 429 errors, like count of errors 😅